### ISSUE TITLE
adding training accuracy to skorch history

### ISSUE THREAD

**Issue by sushmit86** [skorch-dev/skorch#798]
Hi,
I am new to Skorch. This might be an easy one. Is there a way to add training accuracy to the history? For example, right now the training output is something like this.

![image](https://user-images.githubusercontent.com/5600498/129651330-62aa40da-c013-4608-a078-afb841d38f7b.png)
 
Is it possible to add the train_acc. I was thinking of using a callback but it seems like callbacks would work on validation set

**Comment by ottonemo** [comment]
Hi!

You can add an [epoch scoring callback][1] for accuracy on the training data set. Example:

```
net = NeuralNetClassifier(
    # ...
    callbacks=[
        EpochScoring(scoring='accuracy', on_train=True),
    ],
)
```

By default scores are evaluated on the validation set. By setting `on_train=True` the scorer is evaluated on the training set instead.

[1]: https://skorch.readthedocs.io/en/stable/callbacks.html#skorch.callbacks.EpochScoring

**Comment by sushmit86** [comment]
@ottonemo  Thanks so much for the reply
